### PR TITLE
[TSC] Update Principles, Scope, and Relationship with other communities

### PR DESCRIPTION
**Pull request by rabi-abdel**
Fixes https://github.com/cntt-n/CNTT/issues/1432
Fixes https://github.com/cntt-n/CNTT/issues/1335
Fixes https://github.com/cntt-n/CNTT/issues/739

### CODE REVIEW

File: doc/tech/README.md [hunk 1]
```diff
@@ -144,7 +144,8 @@ There will be dedicated OVP hacking tracks to facilitate VNF vendor onboarding a
 
 The CNTT will collaborate with the respective API workgroups of SDOs (ETSI, MEF, TM Forum) as much as possible.  However, to collate on the relevant APIs from these SDOs in some cases requires special permission since information might not be available to the public.  For example. MEF LSO APIs & TM Forum OpenAPIs are accessible by members only.
 
->_More diagrams needed to illustrate relationships_
+<p align="center"><img src="./figures/tech_relation.png" alt="scope" title="Document Types" width="100%"/></p>
```
Comment (CsatariGergely) [inline doc/tech/README.md:147]: - I think we should not show here technology choices, like Airship
- Please provide the source of the figure too :)

File: doc/tech/README.md [hunk 2]
```diff
@@ -130,32 +145,31 @@ The CNTT work is not done in a vacuum.  The intention from the beginning was to
...
 
+<a name="3.4.1"></a>
+### 3.4.1 Relationship to ETSI-NFV
+
 The ETSI NFV ISG is very closely related to the CNTT, in that it is a group that is working on supporting technologies for NFV applications. To facilitate more collaboration as the project matures, the CNTT scope (**Figure 2** above) purposely references certain ETSI NFV reference points, as specified by [ETSI GS NFV 002](https://www.etsi.org/deliver/etsi_gs/NFV/001_099/002/01.02.01_60/gs_NFV002v010201p.pdf).
 
-The CNTT is also closely aligned with OVP, an open source, community-led compliance and verification program that demonstrates the readiness and availability of commercial NFV products and services, including cloud infrastructure and VNFs, using OPNFV. OVP combines open source-based automated compliance and verification testing for multiple parts of the NFV stack specifications established by ONAP, multiple SDOs such as ETSI and GSMA, and the LF Networking End User Advisory Group (EUAG).
+<p align="center"><img src="./figures/tech_relation_etsi.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 4:</b> CNTT Relationship with ETSI NFV.</p>
```
Comment (hdamker) [inline doc/tech/README.md:154]: The text says "CNTT Relationship with ETSI NFV" but the actual picture describes only the scope of ETSI NFV. Is this the right picture?

File: doc/tech/README.md (hunk 1 above)
Comment (pgoyal01) [reply doc/tech/README.md]: @CsatariGergely I think we are mentioning opensource technologies and tools in various places.

File: doc/tech/README.md (hunk 2 above)
Comment (rabi-abdel) [reply doc/tech/README.md]: @hdamker fixed in https://github.com/cntt-n/CNTT/pull/1433/commits/36ad1c9177358d096acab0a711ec5a0781cd1057

File: doc/tech/README.md (hunk 1 above)
Comment (rabi-abdel) [reply doc/tech/README.md]: @CsatariGergely fixed in https://github.com/cntt-n/CNTT/pull/1433/commits/252b7178e8c9e3e41513a70896714dec77fdf563

**Pull request merged**